packages/create-redwood-app/template/api/tsconfig.json

```diff
@@ -10,8 +10,8 @@
     "paths": {
       "src/*": ["./src/*"]
     },
-    "typeRoots": ["../node_modules/@types", "../.redwood"],
+    "typeRoots": ["../node_modules/@types", "./node_modules/@types"],
     "types": ["jest"]
   },
-  "include": ["src"]
-}
\ No newline at end of file
+  "include": ["src", "../.redwood/**/*", "types", "../types",]
```

Comment:
Could we take "types" out for now? We're not 100% using it right, it's optional if a user wants to overwrite? Because I would like to play around with the idea of a top-level folder for both generated and hand-written types.

Reply:
types and "..types" are just there for convenience really. For me its a common pattern I use where I add a "types" folder for things I want to share across my project. But will take out if you prefer!